### ISSUE TITLE
Support for node exec options

### ISSUE THREAD

**Issue by Lbsl** [Miramac/vscode-exec-node#15]
I would have the abilities to pass node exec options when execute the js files such as "--require babel-register".
By the way, it's not a big change, I have finished the logic locally but have no permission to push to the repository.

**Comment by Miramac** [comment]
This option already exists, but you must disable the legacy mode for it.
````json
{
  "miramac.node.legacyMode": false,
  "miramac.node.args": ["--require", "babel-register"]
}
````
I hope that works for you :)

If you want to make changes to the code, please send me a pull request. First create a fork of repo, then send me a pull request:  https://help.github.com/articles/creating-a-pull-request-from-a-fork/

**Comment by Lbsl** [comment]
I have tried this args but it doesn't eork in node 7.9, I checked the logic and found the "miramac.node.args" setting are append to the end of node cli which means after the file name, but the --require option must be placed before the file name.

test.js:
``` javascript
import http from 'http';
console.info('import successfully');
```

result:

![node_options](https://cloud.githubusercontent.com/assets/7035546/25461658/d1cee878-2b1c-11e7-853c-ccdec7808f0b.png)

**Comment by Miramac** [comment]
fix with #16 
thank again!
